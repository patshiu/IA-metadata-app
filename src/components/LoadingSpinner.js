import React from 'react';

const LoadingSpinner = () => {
  return (
    <div className="row">
      <div className="section center">
        <span className="load large"></span>
        <h1>Loading...</h1>
      </div>
    </div>
  );
};

export default LoadingSpinner;
