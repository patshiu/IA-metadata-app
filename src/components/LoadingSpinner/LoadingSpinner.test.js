import React from 'react';
import renderer from 'react-test-renderer';

import LoadingSpinner from './LoadingSpinner';

it('renders loading notice and spinner correctly', () => {
  const tree = renderer.create(<LoadingSpinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
