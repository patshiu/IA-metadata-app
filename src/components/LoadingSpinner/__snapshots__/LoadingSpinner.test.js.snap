// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading notice and spinner correctly 1`] = `
<div
  className="row"
  data-testid="loading-spinner-element"
>
  <div
    className="section center"
  >
    <span
      className="load large"
      role="presentation"
    />
    <h1>
      Loading...
    </h1>
  </div>
</div>
`;
